magenta/models/melody_rnn/melody_rnn_sequence_generator.py

```diff
@@ -34,12 +34,16 @@ class MelodyRnnSequenceGeneratorException(Exception):
 class MelodyRnnSequenceGenerator(magenta.music.BaseSequenceGenerator):
   """Shared Melody RNN generation code as a SequenceGenerator interface."""
 
-  def __init__(self, config, steps_per_quarter=4, checkpoint=None, bundle=None):
+  def __init__(self, config, temperature, steps_per_quarter=4, checkpoint=None,
```

Comment:
Adding another argument here is going to screw up the interfaces. Can you just add it to the config in the melody_rnn_generate binary instead?